### ISSUE TITLE
primeng accessibilty violations

### ISSUE THREAD

**Issue by srujana10499** [primefaces/primeng#18942]
### Describe the bug

The p-select, p-radiobutton, p-iconField, p-menubar, p-progressbar component has an accessibility violation when running the IBM Equal Accessibility Checker: 

1. Form control element <input> has no associated label - [for p-radiobutton]
2. Form control element <input> has no associated label - [for p-password]
3. Form control with "searchbox" role has no associated label
4. Form control element <input> has no associated label - [for p-select]
5. Form control with "progressbar" role has no associated label
6. Table has no headers identified  - [for p-treetable]
7. Complex table does not have headers for each cell properly defined with 'header' or 'scope' - [for p-treetable]
8. The 'id' "pn_id_1172_content" specified for the ARIA property 'aria-controls' value is not valid
9. The ARIA attributes "aria-level" are not valid for the element <li> with ARIA role "menuitem" - [for p-menubar]
10. Multiple elements with "form" landmarks within the same parent region are not distinguished from one another because they have the same "" label

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz-starters-sfmwlzcw.stackblitz.io

### Environment

stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the error:
    - Form control element <input> has no associated label - [for p-radiobutton]
    - Form control element <input> has no associated label - [for p-password]
    - Form control with "searchbox" role has no associated label
    - Form control element <input> has no associated label - [for p-select]
    - Form control with "progressbar" role has no associated label
    - Table has no headers identified  - [for p-treetable]
    - Complex table does not have headers for each cell properly defined with 'header' or 'scope' - [for p-treetable]
    - The 'id' "pn_id_1172_content" specified for the ARIA property 'aria-controls' value is not valid
    - The ARIA attributes "aria-level" are not valid for the element <li> with ARIA role "menuitem" - [for p-menubar]
    - Multiple elements with "form" landmarks within the same parent region are not distinguished from one another because they have the same "" label

### Expected behavior

No accessibility violations are present for the p-dialog component

**Comment by JohnFTitor** [comment]
The specific issue for the menuitem is the aria-level: The ARIA attributes "aria-level" are not valid for the element `<li>` with ARIA role "menuitem"
Just in case someone picks this up
